tests/kafkatest/tests/produce_consume_validate.py

```diff
@@ -32,35 +32,38 @@ def __init__(self, test_context):
         # be overidden by inheriting classes.
         self.producer_start_timeout_sec = 20
 
+        # How long to wait for the consumer to start consuming messages?
+        self.consumer_start_timeout_sec = 60
+
         # How long to delay between the start of the producer and consumer? This
         # is important in the case when the consumer is starting from the end,
         # and we don't want it to miss any messages. The race condition this
         # timeout avoids is that the consumer is still starting after the
         # producer begins producing messages, in which case we will miss the
         # initial set of messages and get spurious test failures.
-        self.delay_between_consumer_and_producer_start_sec = 0
+        self.consumer_init_timeout_sec = 0
```

Comment:
Would this be better as `consumer_alive_timeout_sec`? Also the documentation above needs to be updated.